client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -143,6 +158,26 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 					onExpandedChange={ onExpandedStateChange }
 				/>
 			) }
+
+			{ showEmailForwardLink && (
```

Comment:
It might be worth it to only show this link if the user does not already have forwards.

Reply:
I think we should also continue to show a warning for the case where the user already has email forwards. Unless you disagree, I'd prefer to tackle that in a follow-up PR.

Reply:
I've tackled this specific piece in #59640.